package.json

```diff
@@ -10,24 +10,24 @@
     "node": ">= 8.6.0"
   },
   "dependencies": {
-    "clean-css": "4.2.1",
+    "clean-css": "4.2.3",
     "express": "4.17.1",
     "express-history-api-fallback": "2.2.1",
     "glob-to-regexp": "^0.4.1",
-    "highland": "2.13.4",
+    "highland": "2.13.5",
     "html-minifier": "4.0.0",
     "is-glob": "^4.0.1",
-    "minimalcss": "0.8.2",
-    "mkdirp": "0.5.1",
-    "puppeteer": "^1.8.0",
+    "minimalcss": "0.11.0",
+    "mkdirp": "1.0.4",
+    "puppeteer": "^5.4.1",
     "serve-static": "1.14.1",
-    "sourcemapped-stacktrace-node": "2.1.8"
+    "sourcemapped-stacktrace-node": "git+ssh://github.com/miclovich/sourcemapped-stacktrace-node.git"
```

Comment:
forked this and updated master's packages to have the least security vulnerabilities.